vm/src/builtins/range.rs

```diff
@@ -481,10 +483,40 @@ impl PyValue for PyLongRangeIterator {
 }
 
 #[pyimpl(with(PyIter))]
-impl PyLongRangeIterator {}
+impl PyLongRangeIterator {
+    #[pymethod(name = "__length_hint__")]
```

Comment:
`#[pymethod(magic)]` will work.
`magic` adds dunder to rust function name.

Reply:
Yeah, missed this (rest of the range class uses `name` for the dunder methods and it threw me off).